src/main/scala/wdlTools/eval/Eval.scala

```diff
@@ -545,19 +545,22 @@ case class Eval(paths: EvalPaths,
     // split string into lines and drop all leading and trailing empty lines
     val lines =
       s.split("\r\n?|\n").dropWhile(_.trim.isEmpty).reverse.dropWhile(_.trim.isEmpty).reverse
-    val wsRegex = "^([ \t]*)$".r
-    val nonWsRegex = "^([ \t]*)(.+)$".r
-    val content = lines.foldLeft(Vector.empty[(String, String)]) {
-      case (content, wsRegex(txt))        => content :+ (txt, "")
-      case (content, nonWsRegex(ws, txt)) => content :+ (ws, txt)
-    }
-    if (content.isEmpty) {
+
+    if (lines.isEmpty) {
       ""
     } else {
-      val (whitespace, strippedLines) = content.unzip
-      val colOffset = whitespace.map(_.length).min
+      // split lines into leading whitespace and remaining text
+      val wsRegex = "^([ \t]*)(.*)$".r
+      val (whitespace, strippedLines, colOffsets) = lines.map {
+        case wsRegex(ws, txt) if ws.nonEmpty || txt.nonEmpty => (ws, txt, Some(ws.length))
```

Comment:
cant we remove `if ws.nonEmpty || txt.nonEmpty`. and replace `Some(ws.length)` with `ws.length` and then use `colOffsets.min`?    In this case we wouldnt need second `case`. 

ws.length will always return integer >= 0, so why bother with Some/None?

Reply:
Because in the case of an empty line we want to ignore the whitespace.